samples/TealiumHelper.swift

```diff
@@ -29,12 +29,11 @@ class TealiumHelper : NSObject {
         
             tealium = Tealium(config: config,
                               completion:{ () in
-                #if AUTOTRACKING
-                    self.tealium?.autotracking()?.delegate = self
-                #endif
+                                
```

Comment:
a comment here on whats going on with the delegate module would be helpful especially since we are in the helper